### PR TITLE
Add extension installation instructions

### PR DESCRIPTION
**Pull request by mbareta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,3 +209,12 @@ containers do not support. The template for creating custom content elements
 can be found [here](https://github.com/ExtensionEngine/tailor-element-template)
 while the template for creating custom containers can be found
 [here](https://github.com/ExtensionEngine/tailor-container-template).
+
+### Installing extensions
+
+1. copy (or git clone) extension files in `extensions/content-<elements or containers>/<my-extension-name>`
+2. create index.js file in `extensions/content-<elements or containers>` directory
+3. in the file from the previous step add `module.exports = ['my-extension-name'];`
```
Comment (vladimyr) [inline README.md]: Convert this to code block so it gets proper syntax highlighting. :bowtie:

File: README.md [hunk 2]
```diff
@@ -209,3 +209,12 @@ containers do not support. The template for creating custom content elements
...
+
+Note that `module.exports` is an array and you can add as many extensions and just include their folder names in this array to include all of them.
+After installation, the extension is ready for use and should be listed in `activities.rc` config file.
```
Comment (vladimyr) [inline README.md]: Actually, this is legacy config name and `tailor.config.js` is preferred.

File: README.md (hunk 1 above)
Comment (mbareta) [reply README.md]: @vladimyr please confirm this looks good now :)

Comment (mbareta) [reply README.md]: 🙇

**Pull request merged**